service/src/main/java/uk/gov/hmcts/reform/fpl/service/MessageJudgeService.java

```diff
@@ -37,4 +52,31 @@ protected String buildMessageHistory(String message, String history, String send
 
         return join("\n \n", history, formattedLatestMessage);
     }
+
+    protected String getEmailAddressByRoleType(JudicialMessageRoleType roleType) {
+        if (JudicialMessageRoleType.JUDICIARY.equals(roleType)) {
+            return userService.getUserEmail();
+        } else if (JudicialMessageRoleType.CTSC.equals(roleType)) {
+            return ctscEmailLookupConfiguration.getEmail();
+        }
+
+        return EMPTY;
+    }
+
+    protected String resolveSenderEmailAddress(JudicialMessageRoleType roleType, String defaultEmail) {
```

Comment:
maybe choose a different parameter name to "defaultEmail".

think here the default is what gets filled out on the form?